crypto/fipsmodule/aes/aes_nohw.c

```diff
@@ -381,8 +385,13 @@ static inline void aes_nohw_uncompact_block(
       aes_nohw_uncompact_word((a0 & UINT64_C(0x00000000ffffffff)) | (a1 << 32));
   uint64_t b1 =
       aes_nohw_uncompact_word((a1 & UINT64_C(0xffffffff00000000)) | (a0 >> 32));
+#if defined(OPENSSL_BIG_ENDIAN)
+  CRYPTO_store_u64_le(&out[0], b0);
+  CRYPTO_store_u64_le(&out[8], b1);
+#else
   memcpy(out, &b0, 8);
   memcpy(out + 8, &b1, 8);
+#endif // !OPENSSL_BIG_ENDIAN
```

Comment:
[CRYPTO_store_u64_le](https://github.com/aws/aws-lc/blob/main/crypto/internal.h#L911-L917) now handles both BE and LE correctly. 
```suggestion
  CRYPTO_store_u64_le(&out[0], b0);
  CRYPTO_store_u64_le(&out[8], b1);
```